test/e2e/util_test.go

```diff
@@ -573,8 +621,10 @@ func verifyInternalIngressController(t *testing.T, name types.NamespacedName, ho
 				t.Errorf("failed to close reader for pod %s: %v", clientPod.Name, err)
 			}
 		}()
+		curlPodLogs = ""
```

Comment:
Can this become the declaration?

If, after, processing the scanner loop you can't return as "HTTP/1.0 200 OK" was never matched, then print `curlPodLogs`. The output would correspond to each iteration through the polling loop. Right now we just accumulate and print in one fell swoop.

Reply:
> Can this become the declaration?

You mean `var curlPodLogs string`? Done.

> If, after processing the scanner loop, you can't return as "HTTP/1.0 200 OK" was never matched, then print curlPodLogs. The output would correspond to each iteration through the polling loop. Right now we just accumulate and print in one fell swoop.

So yea I see what your saying, but this happens a lot where 200 OK isn't matched after each scanner loop (usually DNS issues...). I think if we print every time, we are gonna clog up the logs with the same repeated failures. Keep in mind, it's not additive: it scans the whole entire log each poll loop. So, with what you are suggesting, I believe you'd get something like this:
```
log line 1
log line 1
log line 2
log line 1
log line 2
log line 3
log line 1
log line 2
log line 3
log line 4
success!
```
I just chose to store the last log scan and only output on failures to keep the logs clean. Let me know if this makes sense or if I'm not seeing what you are saying.

Reply:
Just for posterity, here's what it currently looks like (I intentionally broke it with an invalid IP):
```
    util_test.go:637: failed to verify connectivity with workload with address: 10.0.128.15 using internal curl client. Curl Pod Logs:
        * Rebuilt URL to: http://test.10.0.128.15/
        * Could not resolve host: test.10.0.128.15
        * Closing connection 0
        * Rebuilt URL to: http://test.10.0.128.15/
        * Could not resolve host: test.10.0.128.15
        * Closing connection 1
        * Rebuilt URL to: http://test.10.0.128.15/
        * Could not resolve host: test.10.0.128.15
        * Closing connection 2
    panic.go:482: deleted ingresscontroller unmanaged-migrated-internal
--- FAIL: TestUnmanagedDNSToManagedDNSInternalIngressController (173.51s)
FAIL
FAIL	github.com/openshift/cluster-ingress-operator/test/e2e	176.610s
FAIL
make: *** [Makefile:56: test-e2e] Error 1

```